src/SIL.LCModel/DomainServices/WritingSystemServices.cs

```diff
@@ -1101,16 +1076,10 @@ public static bool TryWs(LcmCache cache, int wsPreferred, int wsSecondary, int h
 
 
 		/// <summary>
-		/// Try the given ws and return the resulting tss and actualWs.
+		/// Try the given ws (can be magic) and return the resulting tss and actualWs.
 		/// </summary>
-		/// <param name="cache">The cache.</param>
-		/// <param name="ws">The ws.</param>
-		/// <param name="hvoOwner">The hvo owner.</param>
-		/// <param name="flidOwning">The flid owning.</param>
-		/// <param name="actualWs">The actual ws.</param>
-		/// <param name="tssResult">The TSS result.</param>
 		/// <returns>
-		/// true if ws is nonzero and tssResult has content (length greater than 0)
+		/// true if ws is real and tssResult has content (length greater than 0)
 		/// </returns>
 		public static bool TryWs(LcmCache cache, int ws, int hvoOwner, int flidOwning, out int actualWs, out ITsString tssResult)
```

Comment:
I briefly considered moving these up with the other helper method, and would still be glad to if prompted, but figured that would be noisier than some would prefer.